src/snowflake/cli/api/commands/flags.py

```diff
@@ -378,15 +378,21 @@ def like_option(help_example: str):
     )
 
 
+def _pattern_option_callback(value):
+    if value.count("'") != value.count("\\'"):
```

Comment:
```suggestion
    if value and value.count("'") != value.count("\\'"):
```

Currently the following does not work:
```
snow object stage list TEST_STAGE_1 
```

Reply:
and you have to return the value in the callback because now the option does not work

Reply:
Fixed :)